tests/testthat/tests_modelEstimation.R

```diff
@@ -146,6 +146,60 @@ test_that("modelEstimation window params work", {
   
 })
 
+test_that("modelEstimation works for POSIXct and Date classes", {
```

Comment:
I don't think we can say modelEstimation will work with sub-daily timesteps. There are a bunch of steps in the modelEstimation code that assume daily time steps. We'd have to *very very* carefully go through all of that and re-do some logic for sure (we assume 1 row equals 1 day, we can't do that anymore). I would honestly just take these tests out because I think it would only confuse future us.